patches/node/inspector_add_network_initiator_in_inspector_protocol.patch

```diff
@@ -0,0 +1,322 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chengzhong Wu <cwu631@bloomberg.net>
+Date: Wed, 5 Feb 2025 18:03:57 +0000
+Subject: inspector: add Network.Initiator in inspector protocol
```

Comment:
This patch landed in node v22.15.0, so it will be unnecessary when that dependency is bumped.

Reply:
I removed this patch here, since it will get pulled in later. That makes this PR much simpler and easier to backport.